core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -720,7 +715,7 @@ private void loadMetadataForPath(LockedInodePath inodePath)
     }
     LoadMetadataContext ctx = LoadMetadataContext.mergeFrom(
         LoadMetadataPOptions.newBuilder()
-            .setCommonOptions(NO_TTL_OPTION)
+            .setCommonOptions(mSyncOptions)
```

Comment:
I don't think this is the correct way to go about implementing this

This will cause files loaded from `ls` and other operations you listed to gain the TTL for files loaded into alluxio that may have already been created outside of alluxio. Even if that is the desired behavior, it breaks backwards compatbility as far as I am aware.

The file TTL options are only supposed to take effect for files created through alluxio. hence alluxio.user.file.**create**.ttl. Using the TTL from client options is something that should be discussed amongst some other maintainers and would need to be configurable behavior via _another_ configuration. Otherwise, it should wait until a 3.0 release.

When we re-wrote a bunch of the sync code, the behavior you're trying to implement here actually appeared and was labeled as a bug (and then subsequently fixed), so I think this PR needs further discussion.

Reply:
@LuQQiu FYI

Reply:
@ZacBlanco @LuQQiu In other words, TTL only valid when we create new file in alluxio. My usage scenario is in Ad-hoc,  the client only care about data reading, the data automatically expires when the alluxio.user.file.create.ttl is set. @LuQQiu  What do you think？

Reply:
I agree that `create ttl` should be only valid when files are created from Alluxio namespace.
i am not quite sure whether we have something like `load ttl` and whether we want to have something like `load ttl`? @ZacBlanco  do you aware of such things?